sdk/notificationhubs/notification-hubs/src/serializers/registrationSerializer.ts

```diff
@@ -79,7 +87,7 @@ export interface AppleTemplateRegistrationDescription
   /**
    * The expiry date.
    */
-  expiryDate?: string;
+  expiry?: Date;
```

Comment:
I think we use [`expiresOn`](https://github.com/Azure/azure-sdk-for-js/blob/2bb81e90e7d4652cd6ce39ee491722e6c8e93164/sdk/textanalytics/ai-text-analytics/src/models.ts#L970) but not sure if it makes sense here. Also, maybe consider marking output properties as being readonly.

Reply:
Removed expiry as it doesn't make sense anymore